### ISSUE TITLE
Swin-B model performance lower than mmpretrain

### ISSUE THREAD

**Issue by zhaojun060708** [huggingface/transformers#26305]
I tried to train my own dataset using huggingface's transformers and mmpretrain in swin-b model

With the same learning rate and same augmentation pipeline, the performance of huggingface is 4% lower than mmpretrain, any one have met the same with me?

**Comment by NielsRogge** [comment]
Hi,

Which models are you comparing exactly? Did you verify logits match between those models on the same input image?
